earn/src/components/advanced/BorrowMetrics.tsx

```diff
@@ -268,14 +265,34 @@ export function BorrowMetrics(props: BorrowMetricsProps) {
       </MetricsGridUpper>
       <MetricsGridLower>
         <HealthMetricCard health={marginAccount.health || 0} />
-        <HorizontalMetricCard label='Liquidation Distance' value={liquidationDistanceText} />
         <HorizontalMetricCard
           label='Daily Interest Owed'
           value={`${formatTokenAmount(dailyInterest0, 2)} ${marginAccount.token0.symbol},  ${formatTokenAmount(
             dailyInterest1,
             2
           )} ${marginAccount.token1.symbol}`}
         />
+        {mostRecentModifyHash && (
+          <HorizontalMetricCard label='Last Modified'>
+            <Text size='M'>
+              <a className='underline text-purple' rel='noreferrer' target='_blank' href={mostRecentModifyUrl}>
+                {mostRecentModifyTimeStr}
+              </a>{' '}
+              using {mostRecentManagerName ? 'the ' : 'an '}
+              <a className='underline text-purple' rel='noreferrer' target='_blank' href={mostRecentManagerUrl}>
+                {mostRecentManagerName ?? 'unknown manager'}
+              </a>
+            </Text>
+          </HorizontalMetricCard>
+        )}
```

Comment:
Different parts of this will be available at different times making it look a bit weird. It would be nice to add a placeholder of some kind.